app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
+    [elf.play_elf_song() for elf in elves]
```

Comment:
Using a list comprehension solely for its side effects is not recommended. Consider using a simple for loop instead: `for elf in elves: elf.play_elf_song()`.